### PR TITLE
Add check for missing `path` in client `host_volume` config

### PR DESCRIPTION
**Pull request by dttung2905**
Hi team,

This PR aims to add additional check for missing `path` value in client `host_volume` config. I have added 2 more test cases to validate this change. Please let me know if I need to make any additional changes for this :pray: 

Fixes : https://github.com/hashicorp/nomad/issues/16968

### CODE REVIEW

File: command/agent/command.go [hunk 1]
```diff
@@ -379,6 +379,15 @@ func (c *Command) IsValidConfig(config, cmdConfig *Config) bool {
 		return false
 	}
 
+	if !reflect.ValueOf(config.Client.HostVolumes).IsZero() {
```
Comment (lgfa29) [inline command/agent/command.go:382]: Is this condition necessary? If the list `IsZero` then the range below would be skipped anyway right?

**Review by lgfa29 — COMMENTED** [review]
Thanks for the PR @dttung2905!

I left a quick question there just to make sure I'm not missing anything.

File: command/agent/command.go (hunk 1 above)
Comment (dttung2905) [reply command/agent/command.go]: Hi @lgfa29,
Thanks for the super quick review! It is indeed redundant here .

**Review by lgfa29 — APPROVED** [review]
Awesome, thanks for checking that 😄

**Pull request merged**